lightly_studio/src/lightly_studio/resolvers/caption_resolver.py

```diff
@@ -78,3 +78,46 @@ def get_all(
         total_count=total_count,
         next_cursor=next_cursor,
     )
+
+
+def get_by_ids(session: Session, caption_ids: list[UUID]) -> list[CaptionTable]:
+    """Retrieve captions by IDs."""
+    results = session.exec(
+        select(CaptionTable).where(col(CaptionTable.caption_id).in_(caption_ids))
+    ).all()
+    # Return samples in the same order as the input IDs
+    caption_map = {caption.caption_id: caption for caption in results}
+    return [caption_map[id_] for id_ in caption_ids if id_ in caption_map]
+
+
+def update_text(
```

Comment:
The implementation is very different from updating annotations: https://github.com/lightly-ai/lightly-studio/blob/45ba5dfbc0b7aa0647f8025fed3f65ad211a8c08/lightly_studio/src/lightly_studio/resolvers/annotation_label_resolver/update.py#L17

Can we align on one implementation style?